data/src/test/java/org/oppia/android/data/backends/gae/RemoteAuthNetworkInterceptorTest.kt

```diff
@@ -92,51 +89,82 @@ class RemoteAuthNetworkInterceptorTest {
     assertThat(serviceRequest.header("app_version_code")).isNull()
 
     call.execute()
-    val interceptedRequest = mockWebServer.takeRequest(timeout = 8, unit = TimeUnit.SECONDS)
-    verifyRequestHeaders(interceptedRequest!!.headers)
+    val interceptedRequest = mockWebServer.takeRequest(
+      timeout = testCoroutineDispatcher.DEFAULT_TIMEOUT_SECONDS,
+      unit = testCoroutineDispatcher.DEFAULT_TIMEOUT_UNIT
+    )
+    interceptedRequest?.let {
+      verifyRequestHeaders(it.headers)
+    }
```

Comment:
I was thinking more like: verifyRequestedHeaders(it?.headers). The null check should work automatically inside verify since you'll also need to do the safe-call operator & Truth will fail if the resulting expression evaluates to null.

Reply:
Done